pkg/controller/plan/kubevirt.go

```diff
@@ -987,6 +994,9 @@ func (r *KubeVirt) dataVolumes(vm *plan.VMStatus, secret *core.Secret, configMap
 		annotations[AnnDefaultNetwork] = path.Join(
 			r.Plan.Spec.TransferNetwork.Namespace, r.Plan.Spec.TransferNetwork.Name)
 	}
+	if r.Plan.Spec.Warm && r.isOvirt() {
+		annotations[AnnBindImmediate] = "true"
```

Comment:
isn't it needed also for warm migrations and cold migrations to remote OCP clusters from vSphere? in that case, maybe we should set this annotation all the time and document that we don't create DVs when using volume populators (maybe to exclude the flow in which el9 image of the conversion pod is used - because it's better to bind the volume on the node that the conversion pod starts on)

Reply:
yes, indeed. but i also saw openstack is set with https://github.com/kubev2v/forklift/blob/main/pkg/controller/plan/kubevirt.go#L1837-L1839, shouldn't it be only if the source is openstack? what will happen if a user set warm true or tries remote cluster? it will enter the datavolume creation. @bennyz

Reply:
yeah

Reply:
need to block warm migrations or migrations to remote clusters from openstack until they're implemented